net-irc/inspircd/inspircd-3.4.0.ebuild

```diff
@@ -0,0 +1,123 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit toolchain-funcs
+
+DESCRIPTION="Inspire IRCd - The Stable, High-Performance Modular IRCd"
```

Comment:
I think you could remove the `Inspire IRCd - ` part, but don't mind either way.

Reply:
I'll refactor it a bit.